scss/_progress.scss

```diff
@@ -19,12 +19,20 @@
   margin-bottom: $spacer-y;
 }
 .progress[value] {
-  // IE10 uses `color` to set the bar background-color
-  color: #0074d9;
   // Remove Firefox and Opera border
   border: 0;
   // Reset the default appearance
   appearance: none;
+  // Set overall border radius
+  border-radius: $border-radius;
+  // Set overall background
+  background-color: #eee;
+}
+.progress[value]::-ms-fill {
+  background-color: #0074d9;
+  // Remove right-hand border of value bar from IE10+/Edge
+  border: 0;
+  border-radius: $border-radius;
 }
```

Comment:
ah, i see this is not exactly how the webkit one is handled - only adds the rounding to the left AND right if it's a 100% value. i'll get onto this tomorrow...